core/src/main/java/info/openrocket/core/models/atmosphere/AtmosphericConditions.java

```diff
@@ -75,18 +94,54 @@ public void setTemperature(double temperature) {
 		this.modID = new ModID();
 	}
 
+	public double getHumidity() {
+		return humidity;
+	}
+
+	public void setHumidity(double humidity) {
+		if (humidity < 0 || humidity > 1) {
+			throw new IllegalArgumentException("Humidity must be between 0 and 1");
+		}
+		this.humidity = humidity;
+		this.modID = new ModID();
+	}
+
+	/**
+	 * Calculate the saturation water pressure using the Clausius-Clapeyron equation.
+	 * @return The saturation vapor pressure in Pa
+	 */
+	public double vaporPressureSaturation() {
+		return 611.3 * Math.exp(5423 * (1/273.15 - 1/getTemperature()));
```

Comment:
You can precalculate the 1/273.15 division to improve efficiency. I would put the full equation (like you have now) as a comment before the return function and then use this equation:
```java
// 611.3 * Math.exp(5423 * (1/273.15 - 1/getTemperature()));
return 611.3 * Math.exp(19.854 - 5423/getTemperature());
```